chapters/ch07.asciidoc

```diff
@@ -636,9 +636,7 @@ You could polyfill +Math.log10+ using the +Math.LN10+ constant.
 
 [source,javascript]
 ----
-function log10 (value) {
-  return Math.log(x) / Math.LN10
-}
+Math.log10 = value => Math.log(x) / Math.LN10
```

Comment:
Yay for consistency! I would have preferred _not_ extending natives and going with ponyfills by default, but that’s your call :)